demo/react-native-clock/package.json

```diff
@@ -14,10 +14,7 @@
     "ios-bundle": "react-native bundle --dev false --platform ios --entry-file index.js --bundle-output ios/app.jsbundle"
   },
   "dependencies": {
-    "@picovoice/picovoice-react-native": "=2.2.2",
-    "@picovoice/porcupine-react-native": "=2.2.1",
-    "@picovoice/react-native-voice-processor": "=1.2.0",
-    "@picovoice/rhino-react-native": "=2.2.2",
+    "@picovoice/react-native-voice-processor": "^1.2.0",
```

Comment:
we don't need any of the actual dependencies?

Reply:
it's just the way I'm doing the local install/test. I do a yarn pkg on the binding and then a yarn add *.tgz to add it to the demo. I was finding just adding it by path wasn't working properly. For release I'll be adding these back with their version numbers.